apps/src/applab/ApplabVisualizationColumn.jsx

```diff
@@ -12,32 +23,49 @@ var connect = require('react-redux').connect;
 var ApplabVisualizationColumn = React.createClass({
   propTypes: {
     isReadOnlyWorkspace: React.PropTypes.bool.isRequired,
+    hideSource: React.PropTypes.bool.isRequired,
+    isEmbedView: React.PropTypes.bool.isRequired,
+
     isEditingProject: React.PropTypes.bool.isRequired,
     screenIds: React.PropTypes.arrayOf(React.PropTypes.string).isRequired,
     onScreenCreate: React.PropTypes.func.isRequired,
   },
 
   render: function () {
+    var showFrame = !this.props.isShareView;
+
     return (
-      <div id="visualizationColumn">
+      <div
+          id="visualizationColumn"
+          className={classNames({with_padding: this.props.visualizationHasPadding})}
+          style={[
+            (this.props.isEmbedView || this.props.hideSource) && styles.nonResponsive
+          ]}
+      >
```

Comment:
I was viewing the exception in our style guide as meaning you don't have put them on a new line (but you can). I agree that it might be worth removing the exception from the style guide at this point.